name.abuchen.portfolio/src/name/abuchen/portfolio/online/Column.java

```diff
@@ -0,0 +1,166 @@
+    public abstract void setValue(Element value, LatestSecurityPrice price, String languageHint) throws ParseException;
+
+    protected long asQuote(Element value, String languageHint) throws ParseException
+    {
+        String text = value.text().trim();
+
+        DecimalFormat format = null;
+
+        if ("de".equals(languageHint)) //$NON-NLS-1$
+            format = DECIMAL_FORMAT_GERMAN.get();
+        else if ("en".equals(languageHint)) //$NON-NLS-1$
+            format = DECIMAL_FORMAT_ENGLISH.get();
+
+        if (format == null)
+        {
+            // check first for apostrophe
+
+            int apostrophe = text.indexOf('\'');
+            if (apostrophe >= 0)
+                format = DECIMAL_FORMAT_APOSTROPHE.get();
+        }
+
+        if (format == null)
+        {
+            // determine format based on the relative location of the last
+            // comma and dot, e.g. the last comma indicates a German number
+            // format
+            int lastDot = text.lastIndexOf('.');
+            int lastComma = text.lastIndexOf(',');
+            format = Math.max(lastDot, lastComma) == lastComma ? DECIMAL_FORMAT_GERMAN.get()
+                            : DECIMAL_FORMAT_ENGLISH.get();
+        }
+
+        double quote = format.parse(text).doubleValue();
+        return Math.round(quote * Values.Quote.factor());
+    }
+    
+    private Hashtable<String,Integer> symbols = new Hashtable<String,Integer>() {
+        {
+            put("k",       1000);
+            put("K",       1000);
+            put("T",       1000);
+            put("M",    1000000);
+        }
+    };
+
+    protected int asInt(Element elem) throws ParseException
+    {
+        String text = elem.text().trim();
+        int multiplier = 1;
+        double value = -0.1;
+        try
+        {
+            Pattern numberPattern = Pattern.compile("([0-9,.]+)");
+            Matcher numberMatcher = numberPattern.matcher(text);
+            if (numberMatcher.find())
+            {
+                String match = numberMatcher.group(1);
+                int lastDot = match.lastIndexOf(".");
+                int lastComma = match.lastIndexOf(",");
+                if (lastComma > lastDot)
+                {
+                    match = match.replaceAll("\\.", "");
+                    match = match.replace(",", ".");
+                }
+                else
+                {
+                    match = match.replaceAll(",","");
+                    match = match.replaceAll("([0-9])\\.([0-9][0-9][0-9])\\.([0-9][0-9][0-9])\\.([0-9][0-9][0-9])","$1$2$3$4");
+                    match = match.replaceAll("([0-9])\\.([0-9][0-9][0-9])\\.([0-9][0-9][0-9])","$1$2$3");
+                    match = match.replaceAll("([0-9])\\.([0-9][0-9][0-9])","$1$2");
+                }
+                value = Double.valueOf(match);
+            }
+        }
+        catch (Throwable ex)
+        {
+            System.err.println("Uncaught exception - " + ex.getMessage());
```

Comment:
Writing the error to System.err will loose the message (only visible in the Dev environment). You can
* store it and return it via the "errors" Object
* or ignore it at completely.

Generally, I am not writing errors per (attempted) import line, because this will results in possibly hundreds of error messages.

Then there is the question what Exception you expect here. I guess it is the Double.valueOf so rather a NumberFormatException